<ui version="4.0" >
 <class>DialogConsole</class>
 <widget class="QDialog" name="DialogConsole" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Serial Console</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>5</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="ConnectLbl" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>120</height>
        </size>
       </property>
       <property name="title" >
        <string>Output</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>2</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>4</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QTextEdit" name="textEdit_out" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly" >
             <bool>true</bool>
            </property>
            <property name="acceptRichText" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>4</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QPushButton" name="pbClear" >
              <property name="minimumSize" >
               <size>
                <width>108</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Clear Console</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbSaveTo" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Save to File ...</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="cbCRLF" >
              <property name="text" >
               <string>Interpret CR/LF</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>170</height>
        </size>
       </property>
       <property name="title" >
        <string>Input</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>2</number>
        </property>
        <property name="spacing" >
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>4</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QTextEdit" name="textEdit_in" >
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>7</hsizetype>
              <vsizetype>1</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="accessibleName" >
             <string/>
            </property>
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="documentTitle" >
             <string/>
            </property>
            <property name="acceptRichText" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>5</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Wait (ms) on LF</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>500</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignLeading</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbOpen" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>21</height>
               </size>
              </property>
              <property name="text" >
               <string>Open File ...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbSend" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>21</height>
               </size>
              </property>
              <property name="text" >
               <string>Send Data</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbStop" >
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>21</height>
               </size>
              </property>
              <property name="text" >
               <string>STOP Input stream</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbRelaySerialPort" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Relay to Serial Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRelayNetwork" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Relay to Network </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbConfig" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Config...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbClose" >
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0" />
 <resources/>
 <connections>
  <connection>
   <sender>pbClear</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_out</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>444</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel" >
     <x>327</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogConsole</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>459</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel" >
     <x>390</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
